### DISCUSSION TITLE
Social Auth

### DISCUSSION THREAD

**Discussion by pkrakesh** [reset-button/ariadne_django#47] [Q&A] [answered]
Is there any way to integrate social auth with Ariadne and Django?

**Comment by maxmorlocke** [comment, marked as answer]
Nothing out of the box, exactly, but this isn't terribly hard to roll.  You should be able to use any of the standard django libraries and add a decorator very similar to [this](https://github.com/reset-button/ariadne_django/blob/main/ariadne_django/auth/decorators/login_is_required.py) to validate a JWT token, call an API, etc.  It's then up to you if you want to build graphql-facing auth endpoints.

**Reply by pkrakesh** [reply]
Thanks.
